velox/dwio/dwrf/test/E2EFilterTestBase.h

```diff
@@ -273,6 +286,15 @@ class E2EFilterTestBase : public testing::Test {
   int32_t flushEveryNBatches_{10};
   int32_t nextReadSizeIndex_{0};
   std::vector<int32_t> readSizes_;
+
+  // Sequence number of batch for checkOwnership().
+  int32_t batchCounter_;
```

Comment:
Would it make sense to move these into a helper class OwnershipChecker for modularity?